packages/@angular/cli/commands/test.ts

```diff
@@ -14,13 +14,15 @@ export interface TestOptions {
   build?: boolean;
   sourcemap?: boolean;
   progress?: boolean;
+  configFile: string;
 }
 
 
 const TestCommand = EmberTestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
     { name: 'code-coverage', type: Boolean, default: false, aliases: ['cc'] },
+    { name: 'config-file', type: Boolean, default: false, aliases: ['c', 'cf'] },
```

Comment:
Personally, I'd prefer the CLI was consistent with itself.  Especially considering karma may not always be the test runner.

Also, the type should be `'Path'`.